apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/ShardEngineImpl.java

```diff
@@ -510,17 +512,34 @@ private void updateRecovery(ShardRecovery recovery, String processedObject) {
         shardRecoveryDaoJdbc.updateShardRecovery(databaseManager.getDataSource(), recovery);
     }
 
-    private Long exportDerivedTable(TableInfo info, CommandParamInfo paramInfo, Set<String> excludedColumns, int pruningTime) {
+    private Long exportDerivedTable(TableInfo info, CommandParamInfo paramInfo,
+                                    Set<String> excludedColumns, int pruningTime,
+                                    String sortColumn) {
         DerivedTableInterface derivedTable = registry.getDerivedTable(info.getName());
         if (derivedTable != null) {
             if (!info.isPrunable()) {
+                // not prunable table
                 if (excludedColumns == null) {
+                    // no columns to exclude from export
                     return csvExporter.exportDerivedTable(derivedTable, paramInfo.getSnapshotBlockHeight(), paramInfo.getCommitBatchSize());
                 } else {
-                    return csvExporter.exportDerivedTable(derivedTable, paramInfo.getSnapshotBlockHeight(), paramInfo.getCommitBatchSize(), excludedColumns);
+                    // some column(s) are excluded
+                    if (sortColumn != null && !sortColumn.isEmpty()) {
```

Comment:
I think, that better to override getMaxMinValue (should return min/max order_id) in dexOrderTable and use ordinary exportDerivedTable

Reply:
ordinary 'exportDerivedTable' gave incorrect export result due to DB_ID as default sorting column. I switched to new added index by 'height' and use it.

Reply:
I removed 'new index' and reverted that code in latest version.